### PR TITLE
Make tin (powder) autolearn at cooking lvl 4+2, Zinc lvl 3+2, remove of zinc powder, move of migration.json, and other

### PR DESCRIPTION
**Pull request by Lamandus**
tin powder is very hard to come by. Solder is used for a lot of recipes, making it a nasty bottleneck. This compromise makes looking for the books still viable and gives players the option to autolearn it at a quite high cooking level of 6.

### CODE REVIEW

**Comment by Zireael07** [comment]
Chemistry IIRC is represented by high levels of cooking. 4 is fairly good cook but nowhere close to chemistry...

**Comment by Lamandus** [comment]
> Chemistry IIRC is represented by high levels of cooking. 4 is fairly good cook but nowhere close to chemistry...

I kept the cooking level the same. level 6 autolearn is higher in the range of understanding how to make nitric acid, aluminum sulfate, thermite, etc.

**Comment by Zireael07** [comment]
I'd bump both zinc and tin to cooking 5, then

**Comment by Lamandus** [comment]
> I'd bump both zinc and tin to cooking 5, then

yeah, but then lvl 7 cooking for making tin powder is a bit much
I think this is a good compromise.

**Comment by Zireael07** [comment]
I said 5, not 7 - where are you getting 7 from?

**Comment by Lamandus** [comment]
> I said 5, not 7 - where are you getting 7 from?

4+2 -> the recipe is autolearn available at lvl 6. you want to bump it to 5, so it would make sense to bump it to 5+2, because 5+1 for autolearn is a bit weak or near to not worthy.

**Comment by Lamandus** [comment]
Ok, update. 

- Remove of zinc powder, 
- migratiing zinc powder to zinc. 
- Replacing the spwans of zinc powder with just zinc. 
- Lowering the weight of zinc. 
- changing the recipes producing zinc (added in books akin to tin)
- move migration.json to the obsolete folder, since it only involves obsoleted objects.

File: data/json/obsoletion/items.json [hunk 1]
```diff
@@ -4354,5 +4354,24 @@
     "cutting": 3,
     "to_hit": -3,
     "flags": [ "STAB", "SHEATH_KNIFE", "CONDUCTIVE" ]
+  },
+  {
+    "type": "AMMO",
+    "id": "chem_zinc_powder",
```
Comment (Coolthulhu) [inline data/json/obsoletion/items.json:4360]: I don't think you need both an obsolete id and migration at once. Migration alone should suffice.

**Pull request merged**